translate conditional rendering page

jahe opened the pull request:
Hi! 👋

This PR contains the translation of the 'Conditional Rendering' page.
I left the codepen links untouched because I only translated comments and display text into german so that the difference might not have a huge impact on the reader.

If you find that inappropriate there are two possible solutions coming to my mind:
1. Use the original english variants
2. Fork the codepens

ph1p commented on content/docs/conditional-rendering.md:
```diff
@@ -1,30 +1,30 @@
+Mit React können verschiedene Komponenten erstellt werden, die ein gewünschtes Verhalten kapseln. In Abhängigkeit vom State der Anwendung können dann einige davon dargestellt werden.
 
-Conditional rendering in React works the same way conditions work in JavaScript. Use JavaScript operators like [`if`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else) or the [conditional operator](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) to create elements representing the current state, and let React upd ...
+Die Darstellung auf Basis von Bedingungen funktioniert genauso wie Bedingungen in JavaScript. Mit JavaScript Operatoren wie [`if`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Statements/if...else) oder dem [Bedingten Operator](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) kann man Elemente erstellen, die den aktuellen State reprä ...
```
```suggestion
Die Bedingte Darstellung (engl. conditional rendering) in React funktioniert genauso wie Bedingungen in JavaScript. Benutze JavaScript Operatoren wie [`if`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Statements/if...else) oder den [Bedingten Operator](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) um Elemente zu erstellen, welche den aktuellen State widerspiegeln. React aktualisiert dementsprechend die aktuelle Benutzeroberfläche.
```

jahe replied:
I updated the wording based on your suggestion in 3477e21eb78701659438638f2cc632f272f9c9d2

ph1p commented on content/docs/conditional-rendering.md:
```diff
@@ -36,43 +36,43 @@ function Greeting(props) {
+Man kann Elemente auch in Variablen speichern. Das kann hilfreich sein, wenn man einen Teil der Komponente bedingt darstellen und den Rest unverändert lassen möchte.
 
-Consider these two new components representing Logout and Login buttons:
+Stelle dir diese beiden neuen Komponenten vor, die Anmelde- und Abmeldeknöpfe darstellen sollen.
```
```suggestion
Betrachten wir diese beiden neuen Komponenten, welche Anmelde- und Abmeldebuttons darstellen:
```

jahe replied:
I updated the wording based on your suggestion in 3477e21eb78701659438638f2cc632f272f9c9d2

ph1p commented on content/docs/conditional-rendering.md:
```diff
@@ -36,43 +36,43 @@ function Greeting(props) {
+Im unteren Beispiel erstellen wir eine [zustandsbehaftete Komponente](/docs/state-and-lifecycle.html#adding-local-state-to-a-class), die wir `LoginControl` nennen.
 
-It will render either `<LoginButton />` or `<LogoutButton />` depending on its current state. It will also render a `<Greeting />` from the previous example:
+Sie wird je nach aktuellem State entweder einen `<LoginButton />` oder einen `<LogoutButton />` darstellen. Außerdem wird sie immer das `<Greeting />` aus dem vorherigen Beispiel darstellen.
```
```suggestion
Sie wird je nach aktuellem State entweder einen `<LoginButton />` oder einen `<LogoutButton />` darstellen. Außerdem wird sie immer das `<Greeting />` aus dem vorherigen Beispiel anzeigen.
```

jahe replied:
I updated the wording based on your suggestion in 3477e21eb78701659438638f2cc632f272f9c9d2

jahe commented on content/docs/conditional-rendering.md:
```diff
@@ -36,43 +36,43 @@ function Greeting(props) {
+[**Auf CodePen ausprobieren**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+Abhängig vom Wert des `isLoggedIn` Prop, stellt das Beispiel eine der Begrüßungen dar.
```
I updated the wording based on your suggestion in 3477e21eb78701659438638f2cc632f272f9c9d2

jahe commented:
Hi @ph1p, I really appreciate your suggestions. I've updated the PR with your suggested wording and added codepen examples.

Unfortunately the translated codepens don't include CSS like the english ones because .css files currently aren't recognized by the codepen generator.

I checked a couple of other language translation repos how they deal with this problem -> They just don't translate the code examples.

What's your opinion about how we should handle this issue?

ChrisB9 commented:
Translating code examples is more consistent in my opinion, but if it is technically currently not possible, then we should skip it for now and focus on the things that are still open..
